Dockerfile

```diff
@@ -1,8 +1,6 @@
-FROM golang:1.14
-WORKDIR /app
-COPY go.mod go.sum ./
+FROM golang:1.14.2-alpine3.11
+WORKDIR /usr/src/app
+COPY . /usr/src/app
```

Comment:
After workdir we don’t have to specify path for copy "." will work

`COPY . .`

ref - https://docs.docker.com/engine/reference/builder/#workdir

Reply:
fixed